fixtures/stdin

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+'use strict';
+process.stdin.pipe(process.stdout);
```

Comment:
needs a trailing newline

not warned by XO since it doesn't have an extension.